app/assets/stylesheets/blacklight.scss

```diff
@@ -33,6 +33,10 @@ img.no-icon {
   @extend .btn-outline;
 }
 
+.facet-pagination .btn-outline-secondary.active {
+  @extend .btn-primary;
```

Comment:
Nice!  I forgot about `@extend`.  This seems like a much cleaner way to preserve the old styling in the simplest way possible.